spec/requests/api/v1/settings_spec.rb

```diff
@@ -1,39 +1,46 @@
+# frozen_string_literal: true
+
 require 'rails_helper'
 
-describe 'GET api/v1/settings/must_update', type: :request do
+describe 'GET api/v1/settings/must_update' do
+  subject(:endpoint) { get must_update_api_v1_settings_path, params: }
+
   let(:params) do
     {
       device_version: '1.0'
     }
   end
-  subject { get must_update_api_v1_settings_path, params: }
 
   context 'with correct settings' do
-    let!(:setting_version) { create(:setting_version) }
+    before do
+      create(:setting_version)
+    end
 
     it 'return success' do
-      subject
+      endpoint
       expect(response).to be_successful
     end
 
     it 'returns no need to update' do
-      subject
+      endpoint
```

Comment:
I'd prefer keeping `subject`

Reply:
Agree, at least for when we are just declaring it. We could also modify the `ImplicitSubject` cop. I do prefer naming the subject for most of other cases, but not sure if it's the best to enforce it always.

Reply:
I had to change it because the [NamedSubject](https://www.rubydoc.info/gems/rubocop-rspec/1.6.0/RuboCop/Cop/RSpec/NamedSubject) cop. I don't have any preference about it, I would like to hear the opinion of @santib and @JulianPasquale

Reply:
I also feel a bit strange naming the subject. Always felt `subject` is descriptive enough.
That said, I don't have an strong opinion either. I'm actually ok disabling this cop, doesn't sound too critical

Reply:
I don't have a strong opinion either 😅 whatever the team prefers. Or we can just go with Rubocop default, or check what is the preferred style by Standard.rb 🤷‍♂️